makim/cli.py

```diff
@@ -119,8 +121,7 @@ def _get_args():
         nargs='?',
         default=None,
         help=(
-            'Specify the target command to be performed. '
-            '\nOptions are:\n' + '\n'.join(target_help)
+            'Specify the target command to be performed. Options are:\n' + '\n'.join(target_help)
```

Comment:
this line is very long (~86 characters).
please use pre-commit locally to identify and fix this issue:

```
pre-commit install
pre-commit run --all-files
```

remember to check if you have all your dependencies updated: `poetry install`

Reply:
ping me if you have any questions